src/gpu/intel/ocl/gen9_binary.cl

```diff
@@ -19,8 +19,12 @@
 #if IS_PLAIN_LAYOUT
 KERNEL_ATTR
 __kernel void gen9_binary(__global SRC0_DATA_T *src0,
-        __global SRC1_DATA_T *src1, __global DST_DATA_T *dst POST_OP_ARGS,
-        __global float *src0_scale, __global float *src1_scale) {
+        __global SRC1_DATA_T *src1,
+#if IS_SELECT_BINARY
```

Comment:
Better to stick to `IS_TERNARY` and would be great to depend on runtime (`if`) over macros but this is optional.